paddle.Tensor.__sub__(Tensor([0],"float32"), Tensor([0],"float32"), )

W0425 08:14:03.220175 3801917 gpu_resources.cc:119] Please NOTE: device: 0, GPU Compute Capability: 8.9, Driver API Version: 12.2, Runtime API Version: 11.8
W0425 08:14:03.221094 3801917 gpu_resources.cc:164] device: 0, cuDNN Version: 8.8.
[cuda error] paddle.Tensor.__sub__(Tensor([0],"float32"), Tensor([0],"float32"), ) 
 (External) CUDA error(9), invalid configuration argument. 
  [Hint: 'cudaErrorInvalidConfiguration'. This indicates that a kernel launch is requesting resources that can never be satisfied by the current device. Requestingmore shared memory per block than the device supports will trigger this error, as will requesting too many threads or blocks.See cudaDeviceProp for more device limitations.] (at ../paddle/fluid/pybind/eager_functions.cc:1388)
